version/version.go

```diff
@@ -18,5 +18,5 @@ package version
 
 const (
 	// Version is the current version of Gort
-	Version = "0.9.3-dev.3"
+	Version = "0.9.3-dev.2"
```

Comment:
s/dev.2/dev.4

Reply:
I just pushed an image to `getgort/gort:0.9.3-dev.3`, which should make that version work for you. Probably the best thing for you if you need that functionality is to maintain the working version at `0.9.3-dev.3` (the dev versions aren't especially meaningful anyway).